index.html

```diff
@@ -113,6 +132,11 @@
     ["Markov chain"],
     ["4-form"],
     ["7-chain"],
+    ["operator"],
+    ["homeomorphism"],
+    ["colour"],
```

Comment:
I think I'd prefer American english, so "color". (It looks like the math community is as uncertain about the spelling as the rest of the world, but let me know if I'm wrong and math people are more consistent.)